frontend/src/pages/samples/SamplesPage.tsx

```diff
@@ -1,73 +1,134 @@
 import SamplesList from "../../components/SamplesList";
 import {
   DbGapPhenotypeColumns,
-  ReadOnlyCohortSampleDetailsColumns,
-  combinedSampleDetailsColumns,
+  readOnlyAccessSampleColDefs,
+  readOnlyWesSampleColDefs,
+  combinedSampleColDefs,
 } from "../../shared/helpers";
 import { useState } from "react";
-import { Button } from "react-bootstrap";
-import _ from "lodash";
+import { Button, ButtonGroup } from "react-bootstrap";
 import { CustomTooltip } from "../../shared/components/CustomToolTip";
 import InfoIcon from "@material-ui/icons/InfoOutlined";
+import { ColDef } from "ag-grid-community";
+import { DashboardRecordContext } from "../../generated/graphql";
 
-const WES_SAMPLE_CONTEXT = {
-  fieldName: "genePanel",
-  values: [
-    "Agilent_51MB",
-    "Agilent_v4_51MB_Human",
-    "CustomCapture",
-    "IDT_Exome_v1_FP",
-    "IDT_Exome_V1_IMPACT468",
-    "WES_Human",
-    "WholeExomeSequencing",
+const WES_SAMPLE_CONTEXT = [
+  {
+    fieldName: "genePanel",
+    values: [
+      "Agilent_51MB",
+      "Agilent_v4_51MB_Human",
+      "CustomCapture",
+      "IDT_Exome_v1_FP",
+      "IDT_Exome_V1_IMPACT468",
+      "WES_Human",
+      "WholeExomeSequencing",
+    ],
+  },
+];
+
+const ACCESS_SAMPLE_CONTEXT = [
+  {
+    fieldName: "genePanel",
+    values: [
+      "ACCESS129",
+      "ACCESS146",
+      "ACCESS148",
+      "ACCESS-Heme",
+      "ACCESS-HEME-115",
+      "HC_ACCESS",
+      "HC_Custom",
+      "MSK-ACCESS_v1",
+      "MSK-ACCESS_v2",
+      "HC_CMOCH",
+      "CMO-CH",
+    ],
+  },
+  {
+    fieldName: "baitSet",
+    values: [
+      "MSK-ACCESS-v1_0-probesAllwFP",
+      "MSK-ACCESS-v1_0-probesAllwFP_GRCh38",
+      "MSK-ACCESS-v1_0-probesAllwFP_hg19_sort_BAITS",
+      "MSK-ACCESS-v1_0-probesAllwFP_hg37_sort-BAITS",
+      "MSK-ACCESS-v2_0-probesAllwFP",
+      "ACCESS_HEME_MN1",
+      "ACCESS129",
+      "ACCESS146",
+      "ACCESS148",
+      "ACCESS-HEME-115",
+      "CMO-CH",
+      "MSK-CH",
+    ],
+  },
+];
+
+const tabSettings = new Map<
+  string,
+  {
+    columnDefs: ColDef[];
+    sampleContexts?: DashboardRecordContext[];
+  }
+>([
+  [
+    "All",
+    {
+      columnDefs: combinedSampleColDefs,
+      sampleContexts: undefined,
+    },
+  ],
+  [
+    "WES",
+    {
+      columnDefs: readOnlyWesSampleColDefs,
+      sampleContexts: WES_SAMPLE_CONTEXT,
+    },
+  ],
+  [
+    "ACCESS/CMO-CH",
+    {
+      columnDefs: readOnlyAccessSampleColDefs,
+      sampleContexts: ACCESS_SAMPLE_CONTEXT,
+    },
   ],
-};
+]);
```

Comment:
Here, I'm using a Map for quick access of these keys and values below. The syntax for initializing a Map is a bit odd, but that's just how it was designed in JS.

Did not use the simpler JS object because Map preserves order of keys, which helps ensures that we're always showing the "All", "WES", and "ACCESS" tabs in that order when being rendered with `{Array.from(tabSettings.keys()).map(...`